readme.md

```diff
@@ -1,36 +1,81 @@
-# WebGoat.NET version 0.3
+# dotTEST WebGoat.NET Example
 
-## Build status
+This example shows the following dotTEST capabilities:
+
+- Static analysis and flow analysis capabilities
+- Integration with GitHub pipelines via the [Run dotTEST Action](https://github.com/parasoft/run-dottest-action)
+- Integration with Aider to apply static analysis fixes in GitHub pipelines (dotTEST Autofix feature)
+
+## Table of Contents
+
+- [dotTEST capabilities](#dotTEST-capabilities)
+- [About WebGoat.NET project](#WebGoatNET-version-03)
+
+## dotTEST Capabilities
+
+### Static Analysis
+
+Static analysis and flow analysis help you verify code quality and ensure compliance with industry standards, such as CWE or OWASP. Static analysis is a software testing method done by examining the source code without executing it to detect errors, vulnerabilities, and violations of coding standards. Flow analysis refers to the examination and evaluation of data or control flow within a program or system to identify potential issues such as resource leaks, dead code, security vulnerabilities, or performance bottlenecks. 
+See [Parasoft dotTEST User Guide](https://docs.parasoft.com/display/DOTTEST20251) for details regarding static and flow analysis with dotTEST as well as other dotTEST capabilities.
+
+### Run Parasoft dotTEST GitHub Action
+
+The `Run Parasoft dotTEST` action enables you to run code analysis with dotTEST and review analysis results directly on GitHub. To launch code analysis with dotTEST, add the `Run Parasoft dotTEST` action to your GitHub workflow. The following example shows a simple workflow made up of one job "run-dottest-action": 
+
+TODO: Prepare shot yaml example / add link to example
+
+See [Run dotTEST Action @ GitHub Marketplace](https://github.com/marketplace/actions/run-parasoft-dottest) for details regarding configuration and usage.
+See also [Run dotTEST Action project](https://github.com/parasoft/run-dottest-action).
+
+
+### Autofix in CI/CD Using Aider
+
+`DottestAutoFix` is a Python-based script that leverages AI-powered code analysis to automatically fix dotTEST violations in your .NET projects, based on a generated analysis report. Once a fix is applied, the plugin validates it using `dottestcli` and then creates a pull request in your project repository.

```

Comment:
Aider does not create Pull Requestm not even branch. It just commits the code to the current branch. It is user responsibility to create branch and pull request if it is needed.